icon_creator: controller is crashing

On my Ubuntu 18.04 system, the `icon_creator` controller is crashing when processing the image to auto-crop it and remove the background.


Error:
```
Traceback (most recent call last):
  File "icon_creator.py", line 324, in <module>
    background=background, colorThreshold=colorThreshold)
  File "icon_creator.py", line 225, in process_object
    colorThreshold, shadowColor, postfix)
  File "icon_creator.py", line 83, in take_screenshot
    pilImage = autocrop(pilImage)  # cropped at an early stage to save lot of CPU resources.
  File "icon_creator.py", line 74, in autocrop
    assert False, "Impossible to crop image"
AssertionError: Impossible to crop image
WARNING: 'icon_creator' controller exited with status: 1.
```

--- a/scripts/icon_studio/controllers/icon_creator/icon_creator.py
+++ b/scripts/icon_studio/controllers/icon_creator/icon_creator.py
@@ -65,8 +65,9 @@ def take_original_screenshot(camera, directory):
 def autocrop(im):
     """Autocrop an image based on its upperleft pixel."""
     # reference: https://stackoverflow.com/a/48605963/2210777
-    bg = Image.new(im.mode, im.size, im.getpixel((0, 0)))
-    diff = ImageChops.difference(im, bg)
+    rgbImage = im.convert('RGB')  # from RGBA, needed since Pillow 7.1.0
+    bg = Image.new(rgbImage.mode, rgbImage.size, rgbImage.getpixel((0, 0)))
+    diff = ImageChops.difference(rgbImage, bg)
     diff = ImageChops.add(diff, diff, 2.0)
     bbox = diff.getbbox()
     if bbox:

